environments/softlayer/public.yml

```diff
@@ -109,6 +109,7 @@ localsettings:
   TRANSFER_SERVER: 'nginx'
   WS4REDIS_CONNECTION_HOST: "{{ groups.redis.0 }}"
   XFORMS_PLAYER_URL: "http://{{ groups.touchforms.0 }}:4444/"
+  OBFUSCATE_PASSWORD_FOR_NIC_COMPLIANCE: True
```

Comment:
Seems like this should be a domain-level feature flag since it introduces a fair amount of additional complexity for authentication, and any bugs could affect domains that do not need or want this.